### PR TITLE
Ballot fixes event notification section 1

### PR DESCRIPTION
**Pull request by wmaethner**
Fixes to: #202, #236, #204, #203, #159, #137, #96, #95, #247, #207, #206, #205, #231, #232

### CODE REVIEW

File: docs/specification/May2019Ballot.md [hunk 1]
```diff
@@ -150,31 +150,30 @@ hub.callback=https%3A%2F%2Fapp.example.com%2Fsession%2Fcallback%2Fv7tfwuk17a&hub
...
+The notification's `hub.event` and `context` fields inform the subscriber of the current state of the user's session. The `hub.event` is a user workflow event, from the Event Catalog. The `context` is an array of named FHIR resources (similar to [CDS Hooks's context](https://cds-hooks.org/specification/1.0/#http-request_1) field) that describe the current content of the user's session. Each event in the [Event Catalog](#event-catalog) defines what context is expected in the notification. Hubs MAY use the [FHIR _elements parameter](https://www.hl7.org/fhir/search.html#elements) to limit the size of the data being passed while also including additional, local identifiers that are likely already in use in production implementations. Subscribers SHALL accept a full FHIR resource or the [_elements](https://www.hl7.org/fhir/search.html#elements)-limited resource as defined in the Event Catalog.
```
Comment (lbergnehr) [inline docs/specification/May2019Ballot.md]: If the event catalog is moved outside of the spec, perhaps it shouldn't be referred here. I understand if that is not part of this PR though.

Comment (wmaethner) [reply docs/specification/May2019Ballot.md]: @lbergnehr yeah that same thought occurred to me when looking at some of the other issues. I think for this one it is fine and then the PR that handles moving the Event Catalog will handle all the references to it throughout the spec. It'll be a big one.
